### PR TITLE
Making keymaps based on userspace

### PR DESCRIPTION
**Pull request by Rossman360**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c [hunk 1]
```diff
@@ -0,0 +1,191 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: Any particular reason you're doing a second keymap that's almost identical to the first?

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+extern keymap_config_t keymap_config;
```
Comment (noroadsleft) [inline keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: You shouldn't need this line at all.

```suggestion
```

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+#define EE_HANDS
```
Comment (noroadsleft) [inline keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: This line needs to be in `keyboards/keebio/levinson/keymaps/Rossman360_linux/config.h` to work.

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+#define _QWERTY 0
+#define _DEL 1
+#define _FN1 2
+#define _FN2 3
+#define _NUM 4
```
Comment (noroadsleft) [inline keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: You should use an `enum` here:

```c
enum layer_names {
    _QWERTY,
    _DEL,
    _FN1,
    _FN2,
    _NUM,
};
```

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+  WREFRESH,
+  REMCAPS,
+  SPEAK1,
+  SPEAK2,
+  SPEAK3,
+  SPEAK4
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```
Comment (noroadsleft) [inline keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: The indenting on this function could use some love.

File: keyboards/keebio/tragicforce68/keymaps/default/keymap.c [hunk 6]
```diff
@@ -1,34 +1,237 @@
 #include QMK_KEYBOARD_H
 
-#define _QWERTY 0
-#define _FN1 1
+#define _MAC 0
+#define _REV 1
+#define _MOD 2
+#define _DEL 3
+#define _LIN 4
+#define _META 5
```
Comment (noroadsleft) [inline keyboards/keebio/tragicforce68/keymaps/default/keymap.c]: Again, `enum` recommended.

```c
enum layer_names {
    _MAC,
    _FN1,
    _REV,
    _MOD,
    _DEL,
    _LIN,
    _META,
};
```

File: keyboards/keebio/tragicforce68/keymaps/default/keymap.c [hunk 7]
```diff
@@ -1,34 +1,237 @@
...
+  WREFRESH,
+  SMERGE,
+  PMERGE,
+  URIGHT,
+  ULEFT,
+  REMCAPS,
+  COPYCLOSE
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```
Comment (noroadsleft) [inline keyboards/keebio/tragicforce68/keymaps/default/keymap.c]: Indenting again.

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c (hunk 1 above)
Comment (Rossman360) [reply keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: I actually didn't mean to include this file

Comment (Rossman360) [reply keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: hopefully I just got rid of it?

File: keyboards/keebio/tragicforce68/keymaps/default/keymap.c (hunk 6 above)
Comment (Rossman360) [reply keyboards/keebio/tragicforce68/keymaps/default/keymap.c]: arg, or this one

File: keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c (hunk 4 above)
Comment (Rossman360) [reply keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c]: reverted this file to the way it is on master branch. long story of epic git struggle.

**Comment by fauxpark** [comment]
Looks good!

**Pull request merged**